okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -61,14 +61,21 @@
                                     + "which requires debugging.",
                             uncaughtException))
             .setNameFormat("remoting-okhttp-dispatcher-%d")
-            // The Dispatcher must *not* use daemon threads otherwise it would become impossible to make
-            // outgoing network requests in a shutdown hook.
```

Comment:
This comment was incorrect. Requests will still work fine in hooks as long as we wait for our futures. A call being in flight without anything waiting for the result will no longer block shutdown.